src/main/java/sirius/biz/storage/s3/ObjectStore.java

```diff
@@ -349,6 +350,37 @@ public void deleteObject(BucketName bucket, String objectId) {
         }
     }
 
+    /**
+     * Server-side copies an object between two buckets.
+     *
+     * @param sourceBucket   the source bucket containing the object to copy
+     * @param sourceObjectId the object id co copy from the source bucket
```

Comment:
```suggestion
     * @param sourceObjectId the object ID to copy from the source bucket
```
Why does everybody write ID in lowercase? 😄

Reply:
Because I'm not a fan from VW 😛